### ISSUE TITLE
Slowdown of elements selection/unselection when large number of items

### ISSUE THREAD

**Issue by cengizmurat** [PhilippeMarcMeyer/vanillaSelectBox#24]
Hello,

I have a dropdown list of 3.372 elements.
Everything is working fine in v0.51 but when I decided to upgrade to the latest version (currently v0.57), I noted a slowdown of the script : When I check/uncheck elements, I have a mini freeze time, enough to be felt as a human user.
I will downgrade back to v0.51.

v0.51 : 
https://user-images.githubusercontent.com/14607678/102622806-d2c90080-4141-11eb-9d42-966c3985ebe6.mov

v0.57 : 
https://user-images.githubusercontent.com/14607678/102622815-d8264b00-4141-11eb-8694-5d888c412370.mov

**Comment by PhilippeMarcMeyer** [comment]
I added a bad new feature to quickly : a hidden input that holds the
selected items in a comma delimited string... Very bad idea ! I will get
downgrade master to v 0.56 quickly
On 18 Dec 2020 15:01, "Murat Cengiz" <notifications@github.com> wrote:

Hello,

I have a dropdown list of 3.372 elements.
Everything is working fine in v0.51 but when I decided to upgrade to the
latest version (currently v0.57), I noted a slowdown of the script : When I
check/uncheck elements, I have a mini freeze time, enough to be felt as a
human user.
I will downgrade back to v0.51.

v0.51 :
https://user-images.githubusercontent.com/14607678/102622806-d2c90080-
4141-11eb-9d42-966c3985ebe6.mov

v0.57 :
https://user-images.githubusercontent.com/14607678/102622815-d8264b00-
4141-11eb-8694-5d888c412370.mov

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/PhilippeMarcMeyer/vanillaSelectBox/issues/24>, or
unsubscribe
